README.md

```diff
@@ -81,7 +81,7 @@ Parallelization will also cause the resulting domains to be written in an unpred
 * `sslyze` - TLS configuration, using [`sslyze`](https://github.com/nabla-c0d3/sslyze).
 * `analytics` - Participation in an analytics program.
 * `pageload` - Page load and rendering metrics.
-* `a11y` - Accessibility data with the [`pa11y` CLI tool](https://github.com/pa11y/pa11y) via AWS Lambda (requires an AWS account and some additional setup, described further down this document).
+* `a11y` - Accessibility data with the [`pa11y` CLI tool](https://github.com/pa11y/pa11y) (*optionally* via AWS Lambda which, requires an AWS account and some additional setup, described further down this document).
 
```

Comment:
Actually, this PR removes the lambda option. @gbinal, game to update the README?